### PR TITLE
Add ninja to 'Tools you'll need' in README

### PR DESCRIPTION
**Pull request by m00head**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,10 @@ On Linux, the following tools should be installed:
 - [`genhtml`](http://linux.die.net/man/1/genhtml): for creating reports in html
     format from `lcov` output, provided by the `lcov` package on Ubuntu.
 
+On Linux or Windows, the following tool should be installed and available on your path:
```
Comment (dekimir) [inline README.md]: Ninja is just one alternative -- you can also use make.  Consider rewording this to mention CMake generators and link to the CMake [documentation](https://cmake.org/cmake/help/v3.0/manual/cmake-generators.7.html).

**Comment by m00head** [comment]
Moved ninja to optional tools section, and added information about CMake Generators including link to documentation as requested.

**Comment by dekimir** [comment]
Looks good.  @m00head, can you please add yourself to CONTRIBUTORS?

**Comment by m00head** [comment]
@dekimir I would prefer not to add my name and e-mail address to CONTRIBUTORS, especially for such a minor change to the README. Will you be able to submit this change on my behalf?

**Comment by johcor** [comment]
I signed it!

**Comment by dneto0** [comment]
Thanks @johcor for your contribution!  Looks like the CLA bot doesn't see that you have signed the CLA.  Please check that the email on the commit is registered in your GitHub account.
See https://help.github.com/articles/adding-an-email-address-to-your-github-account/ 
If it's not there, then please add it and then reply again in this PR.

**Comment by johcor** [comment]
https://github.com/johcor

**Pull request closed without merging**